bookops_worldcat/_session.py

```diff
@@ -24,6 +25,10 @@ def __init__(
             5,
             5,
         ),
+        total_retries: int = 0,
+        backoff_factor: float = 0,
+        status_forcelist: Optional[List[int]] = [],
```

Comment:
Does raises mypy errors? [Optional docs](https://docs.python.org/3.8/library/typing.html#typing.Optional).
My understanding of `Optional` is that it can be some Python type or None, equivalent to Union[List[int], None]

Reply:
Basically I'm asking if the default value in line 30 should be `None` instead of `[]`. If the latter, `Optional` can be dropped.

Reply:
For some reason this doesn't raise any mypy errors but it seems like it should. It also isn't working how I thought it would if the session is passed `total_retries` without `status_forcelist`. 

I'll fix this in my next PR.

Reply:
thanks for looking into it!